.github/workflows/checks.yml

```diff
@@ -0,0 +1,41 @@
+name: Checks
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
I’ve used `on: [push, pull_request]`, I wonder if there is a reason to filter push runs to only those that target the `master` branch.

Reply:
This is the same as it's in the main file now. IIRC it's because I have been working directly on this repo, not on a fork, and builds for PRs are duplicated (one for push, one for pull request).